### PR TITLE
Add MERGE_IDS subworkflow

### PR DESCRIPTION
**Pull request by itrujnara**
Added a subworkflow that finds IDs which might correspond to the same biological entity based on taxids and protein sequences

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).

### CODE REVIEW

**Review by luisas — APPROVED** [review]
LGTM, lint is failing but once fixed good for me to be merged :)

File: conf/modules.config [hunk 1]
```diff
@@ -86,6 +122,23 @@ process {
...
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+            enabled: params.output_intermediates
+        ]
+    }
+
+    withName: 'CONCAT_FASTA' {
+        publishDir = [
+            path: { "${params.outdir}/sequences" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
```
Comment (JoseEspinosa) [inline conf/modules.config:138]: This one is not an intermediate?
```suggestion
            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
            enabled: params.output_intermediates
```

File: modules/local/make_report.nf [hunk 2]
```diff
@@ -20,7 +15,12 @@ process MAKE_REPORT {
     task.ext.when == null || task.ext.when
 
     script:
-    prefix = task.ext.prefix ?: meta.id
+     // Exit if running this module with -profile conda / -profile mamba
```
Comment (JoseEspinosa) [inline modules/local/make_report.nf]: umm, I think that I have to move also this on proteinfold 😛

**Review by JoseEspinosa — APPROVED** [review]
Nice! Just a small suggestion/question! 🚀

File: conf/modules.config (hunk 1 above)
Comment (itrujnara) [reply conf/modules.config:138]: Not really IMO, this is the file with all ortholog sequences so it's quite relevant

**Comment by itrujnara** [comment]
@nf-core-bot fix linting

File: docs/images/nf-core-reportho_logo_hex_dark.png
Comment (JoseEspinosa) [inline docs/images/nf-core-reportho_logo_hex_dark.png:1]: ❤️

**Review by JoseEspinosa — APPROVED** [review]
New changes also LGTM!

**Pull request merged**